src/main/java/config/Header.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package config;
+
+public class Header {
+    public static final String ACCEPT = "Accept";
+    public static final String CONTENT_TYPE = "Content-Type";
```

Comment:
You don't need to create these static variables. there are already libraries that provide them like: HTTP.CONTENT_TYPE